src/src/Helpers/I2C_Plugin_Helper.cpp

```diff
@@ -0,0 +1,25 @@
+#include "../Helpers/I2C_Plugin_Helper.h"
+
+/***********************************************************************
+ * checkI2CConfigValid_toHtml(taskIndex, onlyCheck)
+ * Check if I2C is correctly configured and usable for this task
+ * taskIndex: will be used in planned enhancements
+ * onlyCheck = true: no html output is generated
+ * Outputs an error message and returns false if not correct
+ **********************************************************************/
+bool checkI2CConfigValid_toHtml(taskIndex_t taskIndex,
+                                bool        onlyCheck) {
```

Comment:
Maybe change `onlyCheck` to `outputToHtml` as I had to read the source to grasp what it does.